test-runner-poc/config.js

```diff
@@ -69,8 +69,8 @@ class NativeConfiguration {
   url(username, password) {
     const url = this.options.url || 'mongodb://%slocalhost:27017/' + this.db;
     // Fall back
-    const auth = username && password ? String(username) + ':' + String(password) + '@' : '';
-    return String(url) + String(auth);
+    const auth = username && password ? `${username}:${password}@` : '';
+    return `${url} ${auth}`;
```

Comment:
Should there be a space between url and auth?

Reply:
yes

Reply:
`String(url) + String(auth)` gave a space between them